test/filters/test_grid_filter.cpp

```diff
@@ -57,6 +57,7 @@ struct EmptyMapStruct {
   bool
   setUp(experimental::TransformFilter<Filter, EmptyMapStruct> const* transform_filter)
   {
+    (void)transform_filter;
```

Comment:
You can also not name the filter and then this line isn't needed anymore. Same for L119

Reply:
Done